### ISSUE TITLE
Comparison to `croaring` python package

### ISSUE THREAD

**Issue by PeterJCLaw** [Ezibenroc/PyRoaringBitMap#76]
Just for completeness it might be worth including https://pypi.org/project/croaring/ (https://github.com/zacharyvoase/croaring.py) in the comparisons in the README?

**Comment by Ezibenroc** [comment]
Good point. We already have https://github.com/sunzhaoping/python-croaring and https://github.com/andreasvc/roaringbitmap but I missed this third repository. I will add it to my benchmark and re-run it when I can find some time.
